src/main/java/gregtech/common/ClientProxy.java

```diff
@@ -154,33 +152,33 @@ public static void addMaterialFormulaHandler(ItemTooltipEvent event) {
 
         // Handles Item tooltips
         if (!(itemStack.getItem() instanceof ItemBlock)) {
-            String chemicalFormula = null;
+            List<String> chemicalFormula = null;
```

Comment:
I think we should change this to be 2 different methods of handling it, since our fluid tooltip is not just a chemical formula anymore, but a general fluid tooltip application